### PR TITLE
Make Item column wider and tax and quantity columns narrower in invoice PDF

### PR DESCRIPTION
**Pull request by cillian**
#### What? Why?

- Closes https://github.com/openfoodfoundation/openfoodnetwork/issues/12304

**Before**

invoice-before

**After**

invoice-after

**Invoice with longer column names - Before**

invoice-before-longer-headers

**Invoice with longer column names - After**

invoice-after-longer-headers

**Invoice with short item name - Before**

invoice-before-short-item

**Invoice with short item name - After**

invoice-after-short-item

#### What should we test?

- Enable the "invoices" feature toggle e.g. run `Flipper.enable(:invoices)` in a Rails console.
- Create an order for an product with a long name on an enterprise with tax.
- View the PDF invoice by clicking _Actions > Print invoice_ on the order in the admin area.

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [x] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [x] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: spec/system/admin/invoice_print_spec.rb [hunk 1]
```diff
@@ -548,8 +548,9 @@
           expect(page).to have_content "($500,150.00 / kg)" # unit price
           expect(page).to have_content "3 1g $416.79 $1,250.37 20.0% $1,500.45"
           # Enterprise fee
-          expect(page).to have_content "#{enterprise_fee.name} fee by $104.35 15.0% $120.00"
-          expect(page).to have_content "coordinator #{user1.enterprises.first.name}"
+          expect(page).to have_content(
+            "#{enterprise_fee.name} fee by coordinator $104.35 15.0% $120.00 #{distributor.name}"
```
Comment (cillian) [inline spec/system/admin/invoice_print_spec.rb:552]: This matches what's in the locale file... https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/locales/en.yml#L3289

**Comment by RachL** [comment]
@cillian Apologies, when I created this issue, work on the new invoice model was prioritized and I forgot to mention in the issue that this work was under the feature toggle "invoices".
Did you work for this PR on the current invoice model (which involves 2 templates) or the new one (under the feature toggle) which has 1 unique template?

**Comment by cillian** [comment]
@RachL No problem, I just updated the new template under the _invoices_ feature toggle. I didn't make any changes to the other two.

**Comment by RachL** [comment]
@cillian perfect, thanks!

**Review by mkllnk — APPROVED** [review]
This makes sense.

File: spec/system/admin/invoice_print_spec.rb (hunk 1 above)
Comment (dacook) [reply spec/system/admin/invoice_print_spec.rb:552]: Hmm, I wonder why the spec passed before, that looks different.

File: spec/system/admin/invoice_print_spec.rb [hunk 2]
```diff
@@ -705,7 +707,11 @@ def convert_pdf_to_page
   temp_pdf = Tempfile.new('pdf')
   temp_pdf << page.source.force_encoding('UTF-8')
   reader = PDF::Reader.new(temp_pdf)
-  pdf_text = reader.pages.map(&:text)
+
+  # Call 'page.runs.map(&:text)' instead of 'page.text' because the latter doesn't return all text,
+  # see https://github.com/yob/pdf-reader/issues/518
+  pdf_text = reader.pages.map { |page| page.runs.map(&:text).join(' ') }
```
[the remark this answers is not in the document]
Comment (dacook) [reply spec/system/admin/invoice_print_spec.rb:713]: Interesting! Perhaps this would be handy as a helper for other PDF tests

**Review by dacook — APPROVED** [review]
Looks good, and great job on testing the variations

**Comment by filipefurtad0** [comment]
Hey @rioug ,

I've checked this PR with the data indicated on the issue (thank you @RachL!); we can see it widens the column item name column on the invoices:

image

I've verified this affects invoices only when the invoices feature is enabled.

Looking good! 
Merging.

**Pull request merged**